.github/workflows/bump-deps.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     - uses: actions/checkout@v4
     - uses: actions/setup-go@v5
       with:
-        go-version: '1.21.3'
+        go-version: ${{ env.GO_VERSION }}
```

Comment:
Seems like GO_VERSION isn't set in the `env` for this workflow?

Reply:
My bad, added it.